deployments/hadoop-yarn/ansible/36-run-benchmark.yml

```
@@ -0,0 +1,84 @@
+#
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2020, ROE (http://www.roe.ac.uk/)
+#
+#     This information is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     This information is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#
+#     You should have received a copy of the GNU General Public License
+#     along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#   </meta:licence>
+# </meta:header>
+#
+
+- name: "Get Zeppelin IP Address"
+  hosts: localhost
+  vars_files:
+    - config/ansible.yml
+    - /tmp/ansible-vars.yml
+    - config/openstack.yml
+
+  tasks:
+
+    - name: "Discover our Zeppelin node and store IP address in temp file"
+      os_server_info:
+        cloud:  "{{ cloudname }}"
+        server: "{{ deployname }}-zeppelin"
+      register:
+        zeppelinnode
+
+    - local_action: copy content={{ zeppelinnode.openstack_servers[0].accessIPv4 }} dest=/tmp/zeppelin_ip.txt
+
+
+
+- name: "Install and run Python benchmark suite"
+  hosts: monitor
+  gather_facts: yes
+  become: yes
+  become_method: sudo
+  vars_files:
+    - config/ansible.yml
+    - /tmp/ansible-vars.yml
+  vars:
+    zepipaddress: "{{ lookup('file', '/tmp/zeppelin_ip.txt') | trim }}"
+
+  tasks:
+
+    - name: "Creating our Zeppelin config file"
+      copy:
+        dest: "/tmp/user.yml"
+        content: |
+          zeppelin_url: http://{{ zepipaddress }}:8080
```

I think this can just use the local hostname.
```suggestion
          zeppelin_url: http://zeppelin:8080
```

---

Commit to address this comment here: https://github.com/wfau/aglais/pull/583/commits/eeeae66c4c64025049ed978c2451da122f868e38